.github/workflows/schools_updater.yml

```diff
@@ -75,13 +75,24 @@ jobs:
           # Extract only the "name" field from each entry and overwrite data.json
           jq -r '.[].name' data.json >> schools.csv
 
-      - name: Commit and push CSV file
-        uses: stefanzweifel/git-auto-commit-action@v5
-        with:
-          commit_message: Update schools CSV file from API data [skip ci]
-          file_pattern: schools.csv
-          commit_user_name: github-actions[bot]
-          commit_user_email: github-actions[bot]@users.noreply.github.com
-          commit_author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
-          skip_fetch: true
-          skip_checkout: true
+      - name: Commit and push CSV file using deploy key
+        env:
+          DEPLOY_KEY: ${{ secrets.UPDATE_SCHOOLS_DEPLOY_KEY }}
+        run: |
+          # Set up SSH
+          mkdir -p ~/.ssh
+          echo "$DEPLOY_KEY" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+
+          # Move the CSV to the desired directory
+          mv school.csv path/to/your/school.csv
```

Comment:
`school.csv` doesn't exist we just need to stage `schools.csv` which is already in the root of the repo.

https://github.com/MLH/mlh-policies/blob/8fe2c2916bc28e0deec29a3bdc15f4dccf2eb198/.github/workflows/schools_updater.yml#L76